### ISSUE TITLE
No output file

### ISSUE THREAD

**Issue by Handymanalan** [YuzhenYe/DGRscan#1]
Hi YuZhen,

This is Alan and I am running DGRscan on a metagenome-assembled genome (>80%) completeness.

I used the command 
**../DGRscan.py -inseq bin.510.fa -summary bin510.summary** 

but there's no ouput file. The log does say **targetregion []** for each individual contigs. Does that mean there's no tr-vr pairs found in the genome?

Cheers and many thanks

Alan

**Comment by Iris7788** [comment]
I have the same question.

**Comment by snayfach** [comment]
If there is no DGR and the program ran to completion, the log should say "Total tr-vr pair: 0"

**Comment by Iris7788** [comment]
你的邮件我已经收到啦，我会尽快查收哒~

**Comment by ChenTianYi99** [comment]
Yes. i have the same question. I can only see "targetregion []" or "find seed" in the log file (can not find "total tr-vr pair: 0"?). 
Blastall gave several homology but no summary file was generated. 

Best
